automation/test.sh

```diff
@@ -273,14 +276,14 @@ spec:
 EOF
   # Run only Windows tests
   ginko_params="$ginko_params --ginkgo.focus=Windows"
-elif [[ $TARGET =~ multus.* ]]; then
+elif [[ $TARGET =~ cnao ]]; then
```

Comment:
So with this we drop the original multus lane?

Reply:
that was not the plan, let me fix it we should run on both.